src/CMakeLists.txt

```diff
@@ -0,0 +1,43 @@
+#   Copyright 2021 Comcast Cable Communications Management, LLC
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+set(PROJ_CPEABS cpeabs)
+
+if (BUILD_YOCTO)
+add_definitions(-DBUILD_YOCTO)
+endif (BUILD_YOCTO)
+
+if (WEBCONFIG_BIN_SUPPORT)
+add_definitions(-DWEBCONFIG_BIN_SUPPORT)
+endif (WEBCONFIG_BIN_SUPPORT)
+
+set(HEADERS ../include/cpeabs.h)
+
+if (WEBCONFIG_BIN_SUPPORT)
+set(SOURCES rdkb/impl.c)
+else()
+set(SOURCES pods/impl.c)
+endif (WEBCONFIG_BIN_SUPPORT)
+
+#Support webcfg lib for backward compatibility
```

Comment:
Remove webcfg typo from all the files and add cpeabs lib etc , check all the files.